llama_agi/pyproject.toml

```diff
@@ -0,0 +1,49 @@
+[tool.poetry]
+name = "llama_agi"
+version = "0.1.0"
+description = "Building AGI loops using LlamaIndex and Langchain"
+authors = []
+license = "MIT"
+readme = "README.md"
+repository = "https://github.com/run-llama/llama-lab/tree/main/llama_agi"
+include = [
+    "LICENSE",
+]
+keywords = ["LLM", "LlamaIndex", "Langchain", "AGI"]
+
+[tool.poetry.dependencies]
+python = ">=3.8.1,<4.0"
+langchain = ">=0.0.141"
+llama-index = ">=0.5.16"

```

Comment:
might be good to explicitly pin (esp if we change API with 0.6.0 release)

Reply:
gotcha!